test/setupVitest.ts

```diff
@@ -55,6 +55,11 @@ if (isVitestJsdom) {
     cb(0);
     return 0;
   };
-
-  globalThis.Element.prototype.getAnimations = () => [];
+} else if (typeof HTMLElement !== 'undefined') {
+  globalThis.originalGetAnimations = HTMLElement.prototype.getAnimations;
+  Object.defineProperty(HTMLElement.prototype, 'getAnimations', {
+    value: undefined,
+    configurable: true,
+    writable: true,
+  });
 }
```

Comment:
For in-browser tests, you'll need to explicitly set it to `undefined`

Reply:
That's not great DX, especially if someone would want to control it per test.

Reply:
Yeah I had to 'undo' it but only for each of the animation tests you made. This is a lot simpler and a better trade-off than requiring a provider for every tree or a prop for every test

Reply:
That can work, as long as we don't set it ourselves in the package source (we could just read for its existence) it shouldn't cause any problems. A combo of checking for `getAnimations` existence (a good idea anyway) to handle the default jsdom scenario, and that variable for browser tests, seems ok.